python/feathub/processors/flink/table_builder/tests/test_kafka_utils.py

```diff
@@ -145,7 +148,11 @@ def test_kafka_sink(self):
         with patch.object(
             self.t_env, "create_temporary_table"
         ) as create_temporary_table, patch.object(table, "execute_insert"):
-            insert_into_sink(self.t_env, table, sink, ("id",))
+
+            placeholder_descriptor: TableDescriptor = OnlineStoreSource(
```

Comment:
Would it be simpler to use `TableDescriptor(name="", keys=["id"])` here?

Reply:
TableDescriptor is an abstract class and thus is not allowed to be instantiated.